packages/dashboard/src/components/DashboardGridLayout/AddCardPanel/AddCard/AddCard.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { IconButton } from 'office-ui-fabric-react/lib/Button';
+import { Image, ImageFit } from 'office-ui-fabric-react/lib/Image';
+import { getStyles } from './AddCard.styles';
+import { IAddCardProps, IAddCardStyles } from './AddCard.types';
+
+export class AddCard extends React.Component<IAddCardProps> {
+  constructor(props: IAddCardProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const getClassNames = classNamesFunction<IAddCardProps, IAddCardStyles>();
+    const classNames = getClassNames(getStyles!);
+    const { title, imageSrc, description, id } = this.props;
+    let imgClassName = classNames.imageWrapper;
+    let textContainerClassName = classNames.textContainer;
+    let addIconClassName = classNames.icon;
+    if (id) {
+      imgClassName = 'addCardImg' + id + ' ' + imgClassName;
+      textContainerClassName = 'addCardText' + id + ' ' + textContainerClassName;
+      addIconClassName = 'addCardIcon' + id + +addIconClassName;
+    }
+    return (
+      <div
+        className={classNames.root}
+        onMouseEnter={this.mouseEnter.bind(this, imgClassName, textContainerClassName, addIconClassName)}
+        onMouseLeave={this.mouseLeave.bind(this, imgClassName, textContainerClassName, addIconClassName)}
+      >
+        <div className={imgClassName}>
+          <Image src={imageSrc} imageFit={ImageFit.cover} width={150} height={100} />
```

Comment:
You might consider adding an `alt` tag for this image to comply with accessibility

https://webaim.org/techniques/alttext/